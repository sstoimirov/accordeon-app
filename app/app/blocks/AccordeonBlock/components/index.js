export * from "./Accordeon";
export * from "./AccordeonItem";
export * from "./Block";
//# sourceMappingURL=index.js.map