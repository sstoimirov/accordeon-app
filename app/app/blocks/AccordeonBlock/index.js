export * from "./components";
export * from "./state";
//# sourceMappingURL=index.js.map