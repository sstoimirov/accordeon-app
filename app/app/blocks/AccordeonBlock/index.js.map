{"version":3,"file":"index.js","sourceRoot":"","sources":["../../../src/blocks/AccordeonBlock/index.ts"],"names":[],"mappings":"AAAA,cAAc,cAAc,CAAC;AAC7B,cAAc,SAAS,CAAC"}