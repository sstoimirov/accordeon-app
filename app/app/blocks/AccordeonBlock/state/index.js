export * from "./AccordeonState";
//# sourceMappingURL=index.js.map