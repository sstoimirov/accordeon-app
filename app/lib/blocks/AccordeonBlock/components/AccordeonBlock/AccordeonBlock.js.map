{"version":3,"file":"AccordeonBlock.js","sourceRoot":"","sources":["../../../../../src/blocks/AccordeonBlock/components/AccordeonBlock/AccordeonBlock.tsx"],"names":[],"mappings":";;;;;;;;;;;;;;;AAAA,6CAA+B;AAC/B,2CAAsC;AACtC,4CAAyC;AAEzC,uCAA6C;AAG7C,IAAa,cAAc,GAA3B,MAAa,cAAe,SAAQ,KAAK,CAAC,aAAqB;IAE3D,YAAY,KAAU;QAClB,KAAK,CAAC,KAAK,CAAC,CAAA;QACZ,IAAI,CAAC,cAAc,GAAG,IAAI,sBAAc,EAAE,CAAC;IAC/C,CAAC;IAED,MAAM;QACF,OAAO,CACH,oBAAC,qBAAS,IAAC,KAAK,EAAE,IAAI,CAAC,cAAc,CAAC,gBAAgB,EAAE,OAAO,EAAE,IAAI,CAAC,cAAc,CAAC,SAAS,GAAI,CACrG,CAAA;IACL,CAAC;CACJ,CAAA;AAZY,cAAc;IAD1B,qBAAQ;GACI,cAAc,CAY1B;AAZY,wCAAc"}