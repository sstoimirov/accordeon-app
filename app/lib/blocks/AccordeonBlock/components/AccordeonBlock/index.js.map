{"version":3,"file":"index.js","sourceRoot":"","sources":["../../../../../src/blocks/AccordeonBlock/components/AccordeonBlock/index.ts"],"names":[],"mappings":";;;;;AAAA,sCAAiC"}