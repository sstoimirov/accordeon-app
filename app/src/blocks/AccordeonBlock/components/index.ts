export * from "./Accordeon";
export * from "./AccordeonItem"
export * from "./AccordeonBlock";
