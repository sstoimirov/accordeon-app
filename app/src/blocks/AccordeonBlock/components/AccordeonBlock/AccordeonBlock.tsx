import * as React from "react";
import { observer } from "mobx-react";
import { Accordeon } from "../Accordeon";

import { AccordeonState } from "../../state";

@observer
export class AccordeonBlock extends React.PureComponent<{}, {}>{
    accordeonState: AccordeonState;
    constructor(props: any) {
        super(props)
        this.accordeonState = new AccordeonState();
    }

    render() {
        return (
            <Accordeon items={this.accordeonState.accordeonContent} onClick={this.accordeonState.clickItem} />
        )
    }
}