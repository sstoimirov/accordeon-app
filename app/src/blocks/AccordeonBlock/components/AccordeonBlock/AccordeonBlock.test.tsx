import "jest"
import * as React from "react";
import { shallow, configure } from 'enzyme';
import { AccordeonBlock } from "./AccordeonBlock";

import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })
describe("Block",()=>{
    it("Renders correctly", ()=>{
        const comp = shallow(<AccordeonBlock />);
        expect(comp).toMatchSnapshot();
    })
})