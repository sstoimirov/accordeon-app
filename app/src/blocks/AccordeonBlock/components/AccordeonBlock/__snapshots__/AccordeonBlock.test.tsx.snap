// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Block Renders correctly 1`] = `ShallowWrapper {}`;
